web/src/components/new/Results.vue

```diff
@@ -202,7 +202,7 @@
               <td class="pl-0">
                 <template v-for="tag in stream.Tags"
                   ><v-hover v-slot="{ hover }" :key="tag"
-                    ><v-chip small
+                    ><v-chip small :color="tags.filter((e) => e.Name == tag)[0].Color"
```

Comment:
i think we can make this nicer by adding a computed property that returns a list of tag object of the stream, e.g.:

tags.filter((t) => stream.Tags.includes(t.Name))

and then iterating over this computed list instead